tests/integration/bytecode/test_cases.py

```diff
@@ -1618,7 +1618,7 @@
             "value": 0,
             "code": "60ff600052602060206001601f600363fffffffffa50602051",
             "calldata": "",
-            "stack": str(int("2c0c45d3ecab80fe060e5f1d7057cd2f8de5e557", 16)),
+            "stack": str("0x2c0c45d3ecab80fe060e5f1d7057cd2f8de5e557"),
```

Comment:
I guess that your previous `str(int(..))` was `str(0x...)` but without the quotes. In python, you can type `0xf` directly and it prints out (it is a) `15`